examples/Ballot.bl

```diff
@@ -0,0 +1,89 @@
+version 1.1
+
+contract Ballot {
+
+    struct Voter {
+        int weight
+        bool voted
+        address delegate
+        int vote
+    }
+
+    struct Proposal {
+        String name
```

Comment:
minor comment, these primitives: int, bool, address are lowercase. But String is uppercase.

Reply:
Value types are lowercase and reference types are uppercase. That's why String is uppercase here.